testing/fetcher.ts

```diff
@@ -39,6 +40,15 @@ interface RequestError {
   error?: string;
 }
 
+function getTemplateReplacementValueForKey(key: string, invocationToken: string): string {
+  return `{{${key}-${invocationToken}}}`;
+}
+
+// Mirrors our utility replaceAll function in `coda` repo.
+function replaceAll(str: string, find: string, replace: string): string {
```

Comment:
Looks like TypeScript has a native `replaceAll`.

Reply:
Yeah this was very confusing to me. I was using that before and getting no type errors, but on running the tests I was getting a `replaceAll is not a function`, so I switched to fill it like we do in `coda`. Any ideas on why that would be the case?

Reply:
Looks like it was added in ES2021, so we'd probably need to change our target in tsconfig.json to use that. I don't know why VSCode and the compiler aren't complaining though, they should presumably be looking at that version.

Fine to leave this as is but I'd add a comment to remove it when we switch the target to ES2021.